packages/quantic/cypress/integration/facets/numeric-facet/numeric-facet.cypress.ts

```diff
@@ -375,4 +376,17 @@ describe('quantic-numeric-facet', () => {
       });
     });
   });
+
+  describe('with a11y', () => {
```

Comment:
probably should put more meaningful title, as `a11y` is not a properties of the facet